### ISSUE TITLE
Timestamp not recognized by Excel in CSV output

### ISSUE THREAD

**Issue by msaron** [sosedoff/pgweb#163]
I exported a table that has a timestamp without timezone column. The JSON exports the data correctly but the CSV timestamp data column is not recognized by Excel.
JSON data output = 
`[{"created_at":"2015-06-22T01:48:14.928Z","pk":103}]`

CSV data output = 
`pk,created_at`
`103,2015-06-22 01:48:14.928 +0000 +0000`

**Comment by sosedoff** [comment]
Sorry but i don't have Excel around, only Google Docs. I made a sample CSV export document that has both date and datetime columns. Try to import it and see if that works. Here's the link: https://dl.dropboxusercontent.com/u/486271/pgweb-1467084910.csv

Im not 100% sure how timestamps should be formatted in CSV files for Excel ingestion, but with a bit of googling it looks like its a standard `yyyy-mm-dd hh:mm:ss`. I pushed a new branch with the fix if you're interested in trying it out.
